live-editing/configs/GridConfigGenerator.ts

```diff
@@ -338,6 +350,36 @@ export class GridConfigGenerator implements IConfigGenerator {
             })
         }));
 
+        configs.push(new Config({
+            component: GridSelectionTemplateExcelComponent,
+            additionalFiles: ["/src/app/grid/services/financialData.ts"],
```

Comment:
Samples uses static data though..  (`import { DATA } from "../../data/customers";`), this shouldn't be needed.